tests/test_ge_helpers.py

```diff
@@ -0,0 +1,72 @@
+from typing import Generator
+from unittest.mock import MagicMock
+
+import pandas as pd
+import pytest
+
+from schematic.models.GE_Helpers import GreatExpectationsHelpers
+from tests.conftest import Helpers
+
+
+@pytest.fixture(scope="class")
+def mock_ge_helpers(
+    helpers: Helpers,
+) -> Generator[GreatExpectationsHelpers, None, None]:
+    """Fixture for creating a GreatExpectationsHelpers object"""
+    dmge = helpers.get_data_model_graph_explorer(path="example.model.jsonld")
+    unimplemented_expectations = ["url"]
+    test_manifest_path = helpers.get_data_path("mock_manifests/Valid_Test_Manifest.csv")
+    manifest = pd.read_csv(test_manifest_path)
```

Comment:
The `get_data_frame` fixture from the conftest file should be used here to be uniform with how manifests are read in elsewhere in the tests